src/components/SpaceProposalVotesListItem.vue

```diff
@@ -0,0 +1,125 @@
+<script lang="ts" setup>
+import { computed, ref } from 'vue';
+import { ExtendedSpace, Proposal, Vote, Profile } from '@/helpers/interfaces';
+import { useIntl } from '@/composables';
+import { shorten, getIpfsUrl } from '@/helpers/utils';
+
+const props = withDefaults(
+  defineProps<{
+    space: ExtendedSpace;
+    proposal: Proposal;
+    vote: Vote;
+    profiles: Record<string, Profile>;
+    hideUsername?: boolean;
+  }>(),
+  {
+    hideUsername: false
+  }
+);
+
+defineEmits(['openReceiptModal']);
+
+const authorIpfsHash = ref('');
+const relayerIpfsHash = ref('');
+
+const titles = computed(() =>
+  props.proposal.strategies.map(strategy => strategy.params.symbol || '')
+);
+
+const { formatCompactNumber } = useIntl();
+</script>
+
+<template>
+  <div class="flex items-center border-t px-3 py-[14px]">
+    <BaseUser
+      :key="vote.voter"
+      :profile="profiles[vote.voter]"
+      :address="vote.voter"
+      :space="space"
+      :proposal="proposal"
+      :hide-username="hideUsername"
+      width-class="w-[110px] min-w-[110px] xs:w-[130px] xs:min-w-[130px] text-left"
```

Comment:
```suggestion
       :width-class="
        hideUsername
          ? 'w-[50px]'
          : 'w-[110px] min-w-[110px] xs:w-[130px] xs:min-w-[130px] text-left'
      "
```
<img width="515" alt="image" src="https://user-images.githubusercontent.com/51686767/227073744-b9d06b9d-0c28-4db7-bb52-1b261461748b.png">